language/de/acp/board.php

```diff
@@ -348,11 +348,15 @@
 
 // Cookie Settings
 $lang = array_merge($lang, array(
-	'ACP_COOKIE_SETTINGS_EXPLAIN'		=> 'Hier legst du die Einstellungen fest, die verwendet werden, um Cookies an die Browser deiner Benutzer zu senden. In den meisten Fällen sollten die Standardwerte ausreichend sein. Führe Änderungen mit Bedacht durch, fehlerhafte Einstellungen könnten deine Benutzer daran hindern, sich anzumelden.',
+	'ACP_COOKIE_SETTINGS_EXPLAIN'		=> 'Hier legst du die Einstellungen fest, die verwendet werden, um Cookies an die Browser deiner Benutzer zu senden. In den meisten Fällen sollten die Standardwerte ausreichend sein. Führe Änderungen mit Bedacht durch, fehlerhafte Einstellungen könnten deine Benutzer daran hindern, sich anzumelden. Wenn du Probleme mit Benutzern hast, die nicht eingeloggt bleiben, besuche die <b><a href="https://www.phpbb.com/support/go/cookie-settings/">phpBB.com Knowledge Base - Fixing incorrect cookie settings</a></b>.',
 
 	'COOKIE_DOMAIN'				=> 'Cookie-Domain',
+	'COOKIE_DOMAIN_EXPLAIN'		=> 'In den meisten Fällen ist die Cookie-Domain optional. Lasse diese leer, wenn du dir unsicher bist.<br /><br /> Falls du dein Board mit anderer Software integriert hast oder mehrere Domains verwendest, dann kannst du die benötigte Cookie-Domain wie folgt bestimmen: Falls du Domainkombinationen wie <i>example.com</i> und <i>forums.example.com</i>, oder auch <i>forums.example.com</i> und <i>blog.example.com</i> verwendest, dann entferne die Subdomains bis du den gemeinsamen Domainteil gefunden hast, in diesem Beispiel <i>example.com</i>. Nun füge einen Punkt vor den gemeinsamen Domain-Namen und füge diesen in das Feld ein. In unserem Beispiel würdest du also <i>.example.com</i> einfügen (beachte den Punkt am Anfang).',
+
 	'COOKIE_NAME'				=> 'Cookie-Name',
+	'COOKIE_NAME_EXPLAIN'		=> 'Dies kann ein beliebiger Wert sein, z.B. eine zufällige Buchstabenkombination. Wenn die übrigen Cookie Einstellungen geändert werden sollte der Cookie-Name ebenfalls geändert werden.',
 	'COOKIE_PATH'				=> 'Cookie-Pfad',
+	'COOKIE_PATH_EXPLAIN'		=> 'Beachte, dass dies immer ein Slash (/) ist, egal wie deine Board-URL aussieht.',
```

Comment:
Das habe ich mich auch gefragt - hier sollten wir vielleicht den Originaltext mal als Bug melden? 

Das macht so in meinen Augen keinen Sinn. In meinen Augen gehört da was hin, das erklärt, dass man den Cookie auf bestimmte Unterordner beschränken kann, wenn man ihn nur im Forum-Ordner sehen will, ansonsten aber auch per / einfach ein domainweit gültiger Cookie gesetzt werden kann.